client/src/utils/date/fromNow.js

```diff
@@ -0,0 +1,5 @@
+import moment from 'moment';
+
+const fromNow = (dateString) => moment(dateString).fromNow();
+
```

Comment:
WOW that was easy.

Reply:
Yes but the moment library is too big, need to remove the not used code when the build is made.